.env.compose

```
@@ -550,7 +550,7 @@ MCP_TRUSTED_PROXIES=
 # WebSocket Transport Settings
 MCP_WS_PORT=3002
 MCP_WS_HOST=0.0.0.0
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MCP_WS_HOST key should go before the MCP_WS_PORT key

(UnorderedKey)